rails/spec/features/admin_edits_user_spec.rb

```diff
@@ -21,7 +21,7 @@
       @teacher2_name = "#{teacher2_user.first_name} #{teacher2_user.last_name}"
       @student_name = "#{student_user.first_name} #{student_user.last_name}"
       @cohort_a_name = "#{project_a.name}: #{cohort_a.name}"
-      login_with_ui_as('admin', 'password')
+      login_as('admin')
```

Comment:
I don't know why `login_with_ui_as` stopped working here even though it works elsewhere in this set of tests. I couldn't figure it out or fix it, so switched it to `login_as` which works.